pkg/proto/datadog/api/README.md

```diff
@@ -0,0 +1,56 @@
+## gRPC: Protobuf and Gateway code generation 
+
+To generate the code for the API you have defined in your `.proto`
+files we will need three different grpc-related packages: 
+
+- protobuf - protoc-gen-go: generates the golang protobuf definitions.
+- grpc-gateway - protoc-gen-grpc-gateway: generates the gRPC-REST gateway  
```

Comment:
Would this be required for message only protos with no services?

Reply:
If there's nothing to generate, the tool won't do anything for protobuf messages not related with any grpc service. This use-case is already addressed correctly by this PR.